src/components/AccountPage/index.js

```diff
@@ -97,12 +99,14 @@ class AccountPage extends PureComponent<Props, State> {
         balance: {
           currency: formatCurrencyUnit(account.unit, account.balance, {
             showCode: true,
+            locale: settings.language,
           }),
           counterValue: formatCurrencyUnit(
             getFiatCurrencyByTicker(counterValue).units[0],
             data.totalBalance,
             {
               showCode: true,
+              locale: settings.language,
```

Comment:
not sure of the rightness of mixing up `locale` and `language` together, I feel that it brings more confusion on an already confusing theme

Reply:
no it should remain "locale" as in the first parameter of `Number.prototype.toLocaleString`. it is supposed to be a locale string like `fr-FR` etc, not only just a "language" (fr / en / ...).

Which means, in the future, it should probably be `${settings.language}-${settings.region}` , unless we have a `settings.locale` that would be this concatenation

Reply:
yeah, my point is not to rename `locale` to `language`, but only the fact that we put *a variable named `language` inside another, named `locale`. If I'm right it's two different things, ye? Like: `fr` and `fr-FR`

Reply:
yeah I think locale includes both "fr" and "fr-FR" , like the region can be omitted per spec. so locale & language are different concept I feel